src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2945,7 +2945,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         <Type>Non-Resx</Type>
       </EmbeddedResource>
     </ItemGroup>
-
```

Comment:
?

Reply:
I should stick to checking out older versions of files rather than fix white space changes manually 😁